### ISSUE TITLE
environment issue

### ISSUE THREAD

**Issue by whwh747** [wuminye/ImplicitGS#4]
Thank you for the outstanding open-source work! When setting up the environment, specifying - pytorch=2.0.1+cu118 throws an error, and Conda cannot find packages like zCurve and pdb. Could you let me know the reason for this?

**Comment by wuminye** [comment]
There are some errors in the yml file because I wrote it manually. Alternatively, you can use the following commands to install these packages:
`
pip install torch==2.0.1 torchvision==0.15.2 torchaudio==2.0.2 --index-url https://download.pytorch.org/whl/cu118
`
`pip install zCurve 
 `
pdb is part of Python’s standard library, so it comes bundled with Python and is ready to use right away.

**Comment by whwh747** [comment]
Thank you for your response. Should we first set up the virtual environment and then install PyTorch? However, setting up the virtual environment fails due to diffgaussian not finding PyTorch.

**Comment by wuminye** [comment]
Yes, PyTorch should be installed in the virtual environment before adding any submodules. I have updated the .yml file and README for your reference.

**Comment by whwh747** [comment]
i will appreciate it!
